executors/stable/config.yml

```diff
@@ -1,6 +1,6 @@
 jtype: StableDiffusionGenerator
 with:
-  batch_size: 1
```

Comment:
why change batchsize here ?

Reply:
Flash attention benefits larger batch sizes more in terms of speed than plain torch cross attention. It's noted above. The increase in speed with batch size 4 is 33% for me, while batch size 1 is just 15%.

Reply:
and what is the memory price to pay?

Reply:
I tried 768x768 (the max allowable size) and it hit around 14 GB -- if consumer GPUs experience issues with that they can drop down batch size, but it seems okay at the default SD sizes of 512x512. SD was trained on 512x512 and struggles with coherency above that size.

Reply:
> 8292MiB
so this is the total of memory used in the forward pass ? Including the weight of the model ? it means if I have 9GB of vram I can run the inferece with batch_size = 4 ? If yes then lets go for it

Reply:
Yes, that is correct.